.eslintignore

```diff
@@ -1 +1,2 @@
+/copyright.js
```

Comment:
Need to ignore the template itself so that it doesn't complain about finding `YEAR` when it is looking for a number like 2020, 2021 etc.